flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/failover/flip1/RestartBackoffTimeStrategyFactoryLoader.java

```diff
@@ -56,20 +56,20 @@ private RestartBackoffTimeStrategyFactoryLoader() {
 	 *
 	 * @param jobRestartStrategyConfiguration restart configuration given within the job graph
 	 * @param clusterConfiguration cluster(server-side) configuration
-	 * @param isCheckpointingEnabled if checkpointing is enabled for the job
+	 * @param checkpointingEnabled if checkpointing is enabled for the job
 	 * @return new version restart strategy factory
 	 */
 	public static RestartBackoffTimeStrategy.Factory createRestartBackoffTimeStrategyFactory(
 			final RestartStrategies.RestartStrategyConfiguration jobRestartStrategyConfiguration,
 			final Configuration clusterConfiguration,
-			final boolean isCheckpointingEnabled) {
+			final boolean checkpointingEnabled) {
```

Comment:
I am following this https://stackoverflow.com/a/5322666

Reply:
Ok. Seems to me it is mostly referring to class fields' name regarding the getter/setter name.
As a variable, I think both the previous and current one are fine since they make sense and do not look weird.

Reply:
For consistency I would prefer having the same naming conventions for boolean setters and static methods. However, since there is no guideline in the coding style, and we are already using both styles [1][2], I'd be also fine to drop this commit.

[1] https://github.com/apache/flink/blob/86c232437f74576fdee3baad11c58f77714269fc/flink-core/src/main/java/org/apache/flink/core/fs/EntropyInjector.java#L112

[2] https://github.com/apache/flink/blob/86c232437f74576fdee3baad11c58f77714269fc/flink-formats/flink-csv/src/main/java/org/apache/flink/formats/csv/CsvRowSerializationSchema.java#L206

Reply:
Ok.

Reply:
Dropped the commit